### PR TITLE
F/data model

### PR DESCRIPTION
**Pull request by Gnwin**
## Blog App : f/data-model

In this milestone, 
- ruby on rails was setup with postgres
- project schema was built

All requirements have been met according to this document:
https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/projects/creating_data_model_project_v1_1.md

### CODE REVIEW

File: db/migrate/20220721182316_create_posts.rb [hunk 1]
```diff
@@ -0,0 +1,13 @@
+class CreatePosts < ActiveRecord::Migration[7.0]
+  def change
+    create_table :posts do |t|
+      t.text :title
+      t.text :text
+      t.timestamps
+      t.integer :comments_counter, default: 0
+      t.integer :likes_counter, default: 0
+
+      t.references :user, null: false, foreign_key: true
```
Comment (AnselemOdims) [inline db/migrate/20220721182316_create_posts.rb:10]: - [ ] The reference should be to `author` while mapping it to the `user` table. Please kindly adjust to meet ERD diagram requirements.

Hint
`t.references :author, foreign_key: { to_table: 'users' }`

Comment (AnselemOdims) [inline db/migrate/20220721182316_create_posts.rb]: - [x] The reference should be to `author` while mapping it to the `user` table. Please kindly adjust to meet ERD diagram requirements.

Hint
`t.references :author, foreign_key: { to_table: 'users' }`

File: db/migrate/20220723061927_create_comments.rb [hunk 2]
```diff
@@ -0,0 +1,11 @@
+class CreateComments < ActiveRecord::Migration[7.0]
+  def change
+    create_table :comments do |t|
+      t.text :text
+      t.references :user, null: false, foreign_key: true
```
Comment (AnselemOdims) [inline db/migrate/20220723061927_create_comments.rb]: - [x] Kindly see above suggestion and make necessary adjustment

File: db/migrate/20220723062711_create_likes.rb [hunk 3]
```diff
@@ -0,0 +1,10 @@
+class CreateLikes < ActiveRecord::Migration[7.0]
+  def change
+    create_table :likes do |t|
+      t.references :user, null: false, foreign_key: true
```
Comment (AnselemOdims) [inline db/migrate/20220723062711_create_likes.rb]: - [x] Kindly see above suggestion and make necessary adjustment

**Review by AnselemOdims — CHANGES_REQUESTED** [review]
### Hi @Gnwin , 
You have been doing a tremendous job so far :ok_hand:
There are some issues that you still need to work on to go to the next project but you are almost there!

## To Highlight 🎉
PR has a title and summary. :heavy_check_mark:
No linter errors :heavy_check_mark:

## Required Changes ♻️
- [ ] Please also check inline comments under the review

## Optional suggestions
_Comments with the [OPTIONAL] prefix are not a must to implement however, It's strongly recommended to work on them as they can make your code better._

## Happy coding!👏👏👏

#### Please leave any questions or comments in the PR if you have any doubts or unclarity! You can tag me using @AnselemOdims for a quicker response

**_Note: Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

File: db/migrate/20220721182316_create_posts.rb (hunk 1 above)
Comment (Gnwin) [reply db/migrate/20220721182316_create_posts.rb]: author is user

Comment (Gnwin) [reply db/migrate/20220721182316_create_posts.rb]: done.

**Review by arslanbisharat — APPROVED** [review]
Hi @Gnwin,
Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights

- The Pull Request has a proper title and description. ✔️ 
- You have added a descriptive Readme file. Good Job. ✔️ 
- There is no linter error is present. ✔️ 
- Project has met all the technical aspects you needed for this project. Keep rocking! ✔️ 


## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**